klasses/views.py

```diff
@@ -16,9 +16,14 @@ class BootcampViewSet(ListModelMixin, GenericViewSet):
 
     def get_queryset(self):
         """Make a queryset which optionally shows what runs are available for enrollment"""
-        queryset = BootcampRun.objects.all().select_related("bootcamp")
+        user = self.request.user
+        has_enrollments = BootcampRunEnrollment.objects.filter(user=user).exists()
```

Comment:
This is an edge case but we need to add this one last filter condition. I just tested this and it works as expected

```suggestion
        has_enrollments = BootcampRunEnrollment.objects.filter(user=user, active=True).exists()
```